user/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls    import path, include
+from user.views     import UserView
+
+urlpatterns = [
+        path('/signup/initial', UserView.as_view()),
+        path('/signup/final', UserView.as_view()),
```

Comment:
```suggestion
        path('/signup', UserView.as_view()),
```
서로 메소드가 다르기 때문에 이미 분리되어 있으므로, url 두개로 사용하지 않으셔도 됩니다